### ISSUE TITLE
model_dropout

### ISSUE THREAD

**Issue by smita181298** [juan-csv/face_liveness_detection-Anti-spoofing#3]
What is the architecture of the model for emotion detection(model corresponding to model_dropout.hdf5) or how does it work.
@juan-csv .

**Comment by juan-csv** [comment]
hi, this is the architecture of the code model, then I will upload a script to retrain the emotion detection model.

# instantiate model
model = Sequential()

# 1 - Convolution
model.add(Conv2D(64,(3,3), padding='same', input_shape=(48, 48,1)))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 2nd Convolution layer
model.add(Conv2D(128,(5,5), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 3rd Convolution layer
model.add(Conv2D(512,(3,3), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# 4th Convolution layer
model.add(Conv2D(512,(3,3), padding='same'))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))

# Flattening
model.add(Flatten())

# Fully connected layer 1st layer
model.add(Dense(256))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(Dropout(0.25))

# Fully connected layer 2nd layer
model.add(Dense(512))
model.add(BatchNormalization())
model.add(Activation('relu'))
model.add(Dropout(0.25))

model.add(Dense(num_classes, activation='softmax'))

model.summary()

**Comment by smita181298** [comment]
Hello,Thank you so much @juan-csv  for the detailed reply.
Can you share what are the baseline models in case of emotion detection and what to refer to find more about that
1. model_baseline_v1
2. model_v1
3. model_baseline_dropout
4. model_dropout
5. model_baseline_tf_learning
6. model_tf_learning

Again ,Thanks a lot.

**Comment by juan-csv** [comment]
At this moment I do not have enough time to organize the information about the architecture of the other models, for now you can go to https://www.kaggle.com/ashishpatel26/facial-expression-recognitionferchallenge/kernels and review other implementations for the detections of emotions.

**Comment by smita181298** [comment]
Sure and thanks @juan-csv .
